sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 ## 4.2.0-beta.3 (Unreleased)
 
 
+## 4.1.4 (2021-01-15)
```

Comment:
The readmes should reflect the patch/stable release version.

Reply:
What do you mean?